experimental/packages/opentelemetry-exporter-prometheus/test/PrometheusSerializer.test.ts

```diff
@@ -284,20 +284,51 @@ describe('PrometheusSerializer', () => {
       const pointData = metric.dataPoints as DataPoint<number>[];
       assert.strictEqual(pointData.length, 1);
 
-      const result = serializer.serializeSingularDataPoint(metric.descriptor.name, metric.descriptor.type, pointData[0]);
-      return result;
+      if (exportAll) {
+        const result = serializer.serialize(resourceMetrics);
+        return result;
+      } else {
+        const result = serializer.serializeSingularDataPoint(metric.descriptor.name, metric.descriptor.type, pointData[0]);
+        return result;
+      }
     }
 
+    it('should export unit block when unit of metric is given', async () => {
```

Comment:
I think moving this test to the section "serialize an instrumentation metrics" (method `_serializeScopeMetrics`) would be easier since that section is focused on testing serialized metric instrumentation. Also, it can cover more types of aggregations.

This section is testing specifically this statement in the spec:
> Sum metric points MUST have _total added as a suffix to the metric name.

Reply:
+1

Reply:
Thanks good point, I will make this change